README.md

```diff
@@ -22,7 +22,7 @@
 <h2 align="center">Install</h2>
 
 ```bash
-npm i -D uglifyjs-webpack-plugin
+npm i -D uglifyjs-webpack-plugin3
```

Comment:
```
npm i -D uglifyjs-webpack-plugin@beta
```

Reply:
```shell
npm i -D uglifyjs-webpack-plugin@beta
```
gives me `uglifyjs-webpack-plugin@1.0.0-beta.2` still. Is that expected?
```shell
npm i -D uglifyjs-webpack-plugin@1.0.0-beta.3
```
does not work.